### PR TITLE
nbgl: Add centered tag value feature

### PR DESCRIPTION
**Pull request by xchapron-ledger**
## Description

nbgl: Add centered tag value feature

## Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [x] New feature (non-breaking change that adds functionality)
- [x] Breaking change (change that is not backwards-compatible and/or changes current functionality)
- [ ] Tests
- [ ] Documentation
- [ ] Other (for changes that might not fit in any category)

## Breaking changes

Renaming of `force_page_start` into `forcePageStart`, but most don't use it.

### CODE REVIEW

**Review by cedelavergne-ledger — APPROVED** [review]

File: lib_nbgl/include/nbgl_use_case.h [hunk 1]
```diff
@@ -223,7 +223,7 @@ void nbgl_useCaseReviewStreamingFinish(const char           *finishTitle,
 uint8_t nbgl_useCaseGetNbTagValuesInPage(uint8_t                          nbPairs,
                                          const nbgl_layoutTagValueList_t *tagValueList,
                                          uint8_t                          startIndex,
-                                         bool                            *tooLongToFit);
+                                         bool                            *flag);
```
Comment (tdejoigny-ledger) [inline lib_nbgl/include/nbgl_use_case.h:226]: could be good to have a name more explicit for this param

File: lib_nbgl/src/nbgl_use_case.c [hunk 2]
```diff
@@ -1374,18 +1391,20 @@ static void bundleNavReviewStreamingChoice(bool confirm)
...
+ * @param flag (output) set to true if the tag/value needs a specific display:
+ *        - centeredInfo flag is enabled
+ *        - the tag/value doesn't fit in a page
  * @return the number of tag/value pairs fitting in a page
  */
 uint8_t nbgl_useCaseGetNbTagValuesInPage(uint8_t                          nbPairs,
                                          const nbgl_layoutTagValueList_t *tagValueList,
                                          uint8_t                          startIndex,
-                                         bool                            *tooLongToFit)
+                                         bool                            *flag)
```
Comment (tdejoigny-ledger) [inline lib_nbgl/src/nbgl_use_case.c:1402]: same comment as above

File: lib_nbgl/include/nbgl_use_case.h (hunk 1 above)
Comment (xchapron-ledger) [reply lib_nbgl/include/nbgl_use_case.h:226]: Do you have any recommendation for that name?

Comment (tdejoigny-ledger) [reply lib_nbgl/include/nbgl_use_case.h:226]: `requireSpecificDisplay` or
`needSpecificDisplay` or 
`isSpecificDisplayRequired`

for example

Comment (xchapron-ledger) [reply lib_nbgl/include/nbgl_use_case.h]: Done

**Review by nroggeman-ledger — APPROVED** [review]

**Pull request merged**